lib/entities/console.rb

```diff
@@ -14,105 +15,147 @@
          "describe_diff": 'difficult', "user_answer": 'user_answer',
          "wrong_choice": 'wrong_choice', "name": 'name',
          "win": 'win', "failure": 'failure',
-         "restart?": 'restart?', "save?": 'save?' }.freeze
+         "restart?": 'restart?', "save?": 'save?',
+         "statistics": 'statistics', "game_attemt": 'game_attemt',
+         "game_hint": 'game_hint', "continue?": 'continue?' }.freeze
 
 class Console
-  include Storage
-  include Validation
+  include Load
+  include Validate
 
   def initialize(send_to_console = 'greeting')
-    print I18n.t(send_to_console).colorize(:light_blue)
+    print I18n.t(send_to_console)
   end
 
-  def question
-    print yield.colorize(:light_green)
-    gets.chomp
+  def answer_for_user(answer)
+    puts answer
   end
 
-  def what_next?
-    question { '' }
+  def question
+    print yield
+    gets.chomp
   end
 
-  def answer_for_user(answer)
-    puts "Secret code has #{answer}"
-  end
-
-  def choice(user_choice)
-    case user_choice
-    when MENU[:no] then goodbye
-    when MENU[:yes] then choose_the_command
-    when MENU[:game_rules] then rules
-    when MENU[:stats] then stats
-    when MENU[:game_start] then start
-    when MENU[:exit] then goodbye
-    else
-      puts I18n.t(MENU[:wrong_choice]).colorize(:red)
-      choose_the_command
+  def game_over(s_code, _game_statistics, game_status = 'failure')
```

Comment:
why _game_statistics starts with underscore?

Reply:
`rubocop -a `do this changes himself